client/py/examples/control/abort/simulated_daq.py

```diff
@@ -0,0 +1,123 @@
+#  Copyright 2024 Synnax Labs, Inc.
+#
+#  Use of this software is governed by the Business Source License included in the file
+#  licenses/BSL.txt.
+#
+#  As of the Change Date specified in that file, in accordance with the Business Source
+#  License, use of this software will be governed by the Apache License, Version 2.0,
+#  included in the file licenses/APL.txt.
+
+import math
+
+import synnax as sy
+import time
+
+client = sy.Synnax()
```

Comment:
should we add some comments here based on changes someone might want to make to the example? (e.g. put a comment indicating where theyd add stuff if they wanted to increase the num of valve channels or sesnors). but be obvious from the code but also if a prop/test engineer looks at this might be helpful to overcommunicate

Reply:
+1 on this